### PR TITLE
Release notes for 2.10

### PR DESCRIPTION
**Pull request by alvaroaleman**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Documentation**:

**Release note**:

```release-note
none
```

/hold

### CODE REVIEW

File: CHANGELOG_RAW.json [hunk 1]
```diff
@@ -0,0 +1,917 @@
+{
```
Comment (alvaroaleman) [inline CHANGELOG_RAW.json]: This is the raw data I used to create the PR (gathered via a slightly patched version of https://github.com/kubernetes/release/blob/master/cmd/release-notes/). Should we version it here as well or should I remove it?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,86 @@
+### [v2.10.0]()
+
+## Features
+
+### Kubermatic core
+
+* TODO Add docs link:  ACTION REQUIRED: The config option `Values.kubermatic.rbac` changed to `Values.kubermatic.masterController` [#3051](https://github.com/kubermatic/kubermatic/pull/3051) ([@zreigz](https://github.com/zreigz))
```
Comment (mrIncompetent) [inline CHANGELOG.md]: Having a TODO in a release note seems wrong.
@zreigz

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,86 @@
...
+* The user cluster controller manager was added. It is deployed within the cluster namespace in the seed and takes care of reconciling all resources that are inside the user cluster
+* Add feature gate to enable etcd corruption check [#2460](https://github.com/kubermatic/kubermatic/pull/2460) ([@mrIncompetent](https://github.com/mrIncompetent))
+* Kubernetes 1.10 was removed as officially supported version from Kubermatic as its EOL [#2712](https://github.com/kubermatic/kubermatic/pull/2712) ([@alvaroaleman](https://github.com/alvaroaleman))
```
Comment (xrstf) [inline CHANGELOG.md]: **it's**

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,86 @@
...
+* Updated AWS instance types. [#1122](https://github.com/kubermatic/dashboard-v2/pull/1122) ([@maciaszczykm](https://github.com/maciaszczykm))
+* Fixed display number of replicas if the field is empty (0 replicas). [#1126](https://github.com/kubermatic/dashboard-v2/pull/1126) ([@maciaszczykm](https://github.com/maciaszczykm))
+* Added an option to include custom links into the application. [#1131](https://github.com/kubermatic/dashboard-v2/pull/1131) ([@maciaszczykm](https://github.com/maciaszczykm))
+* Remove AWS instance types t3.nano & t3.micro as they are too small to schedule any workload on them [#1138](https://github.com/kubermatic/dashboard-v2/pull/1138) ([@mrIncompetent](https://github.com/mrIncompetent))
+* Redesigned the application sidebar. [#1173](https://github.com/kubermatic/dashboard-v2/pull/1173) ([@maciaszczykm](https://github.com/maciaszczykm))
+
+### Logging & Monitoring stack
+
+* Update fluent-bit to 1.0.6 [#3222](https://github.com/kubermatic/kubermatic/pull/3222) ([@xrstf](https://github.com/xrstf))
+* Update to Prometheus 2.7.1 & Alertmanager 0.16.1 [#2728](https://github.com/kubermatic/kubermatic/pull/2728) ([@xrstf](https://github.com/xrstf))
```
Comment (xrstf) [inline CHANGELOG.md]: This can be removed alltogether because of a later release note entry.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,86 @@
...
+* Update Elasticsearch to 6.6.2 [#3062](https://github.com/kubermatic/kubermatic/pull/3062) ([@xrstf](https://github.com/xrstf))
+* Add Grafana dashboards for kubelet metrics [#3081](https://github.com/kubermatic/kubermatic/pull/3081) ([@xrstf](https://github.com/xrstf))
+* Prometheus was updated to 2.8.1 (Alertmanager 0.16.2), Grafana was updated to 6.1.3 [#3163](https://github.com/kubermatic/kubermatic/pull/3163) ([@xrstf](https://github.com/xrstf))
+* Alertmanager PVC size is configurable [#3199](https://github.com/kubermatic/kubermatic/pull/3199) ([@kron4eg](https://github.com/kron4eg))
+
+### Other
+
+* ACTION REQUIRED: update from Ark 0.10 to Velero 0.11 [#3077](https://github.com/kubermatic/kubermatic/pull/3077) ([@xrstf](https://github.com/xrstf))
+* Replace hand written go tcp proxy with Envoy within the nodeport-proxy [#2916](https://github.com/kubermatic/kubermatic/pull/2916) ([@mrIncompetent](https://github.com/mrIncompetent))
+* It is now possible to configure tolerations and nodeSelectors for the cert-manager deployments [#2879](https://github.com/kubermatic/kubermatic/pull/2879) ([@alvaroaleman](https://github.com/alvaroaleman))
```
Comment (xrstf) [inline CHANGELOG.md]: This can be removed due to the reference to PR #3155 further down.

File: CHANGELOG.md (hunk 2 above)
Comment (alvaroaleman) [reply CHANGELOG.md]: Yeah not sure, problem is the docs are unpublished so I don't know the link. Maybe just remove the TODO and add the link once we export the changelog to docs.kubermatic.io?

Comment (zreigz) [reply CHANGELOG.md]: there is no TODO in my PR description

Comment (alvaroaleman) [reply CHANGELOG.md]: I added the TODO. We will add the docs link when exporting this into docs.kubermatic.io

**Comment by alvaroaleman** [comment]
@zreigz @p0lyn0mial Should there be another release note for `viewers now correctly lost the privilege to edit NodeDeployments`? Is there anything else missing here?

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,84 @@
...
+* Alertmanager PVC size is configurable [#3199](https://github.com/kubermatic/kubermatic/pull/3199) ([@kron4eg](https://github.com/kron4eg))
+
+### Other
+
+* ACTION REQUIRED: update from Ark 0.10 to Velero 0.11 [#3077](https://github.com/kubermatic/kubermatic/pull/3077) ([@xrstf](https://github.com/xrstf))
+* Replace hand written go tcp proxy with Envoy within the nodeport-proxy [#2916](https://github.com/kubermatic/kubermatic/pull/2916) ([@mrIncompetent](https://github.com/mrIncompetent))
+* cert-manager was updated to 0.7.0, Dex was updated to 2.15.0,Minio was updated to RELEASE.2019-04-09T01-22-30Z [#3163](https://github.com/kubermatic/kubermatic/pull/3163) ([@xrstf](https://github.com/xrstf))
+* update nginx-ingress-controller to 0.24.1 [#3200](https://github.com/kubermatic/kubermatic/pull/3200) ([@xrstf](https://github.com/xrstf))
+
+## Bugfixes
```
Comment (toschneck) [inline CHANGELOG.md:88]: I think not all list items are really bugs, some like `VPA is not default deployed anymore` are more Improvements. 
In my opinion bugfixes should be only called like this, when a certain error where in it.
We could also rename it in `Bugfixes / Improvements`

**Review by toschneck — COMMENTED** [review]
one small comment

**Comment by zreigz** [comment]
> viewers now correctly lost the privilege to edit NodeDeployments

@alvaroaleman where is it?

**Comment by alvaroaleman** [comment]
@zreigz It is not there, it was a question if I should add it

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,82 @@
...
+* Initial node deployments are now created inside the same API call as the cluster, fixing spurious issues where the creation didn't happen [#2989](https://github.com/kubermatic/kubermatic/pull/2989) ([@maciaszczykm](https://github.com/maciaszczykm))
+* Errors when reconciling MachineDeployments and MachineSets will now result in an event on the object [#2923](https://github.com/kubermatic/kubermatic/pull/2923) ([@alvaroaleman](https://github.com/alvaroaleman))
+* Filter out not valid VM types for azure provider [#2736](https://github.com/kubermatic/kubermatic/pull/2736) ([@zreigz](https://github.com/zreigz))
+* Mark cluster upgrades as restricted if kubelet version is incompatible. [#2976](https://github.com/kubermatic/kubermatic/pull/2976) ([@maciaszczykm](https://github.com/maciaszczykm))
+* Enable automatic detection of the OpenStack BlockStorage API version within the cloud config [#3112](https://github.com/kubermatic/kubermatic/pull/3112) ([@mrIncompetent](https://github.com/mrIncompetent))
+
+## Dashboard
+
+* It is now possible to edit the project name in UI [#1003](https://github.com/kubermatic/dashboard-v2/pull/1003) ([@kgroschoff](https://github.com/kgroschoff))
```
Comment (alvaroaleman) [inline CHANGELOG.md:38]: @maciaszczykm Should we say something about the redesign? Are there other things that were done but have no release note?

**Comment by zreigz** [comment]
> @zreigz It is not there, it was a question if I should add it

fine for me
cc @p0lyn0mial WDYT?

File: CHANGELOG.md (hunk 7 above)
Comment (maciaszczykm) [reply CHANGELOG.md:38]: > Redesigned the application sidebar. [#1173](https://github.com/kubermatic/dashboard-v2/pull/1173) ([@maciaszczykm](https://github.com/maciaszczykm))

I think that fragment touches the redesign topic well. We can add something about changing the application colors. But from the redesign, we have done only the sidebar and footer.

Comment (alvaroaleman) [reply CHANGELOG.md:38]: I think we should say something about the colors. What about `The color theme of the Dashboard got redesigned`?

**Comment by p0lyn0mial** [comment]
> @zreigz It is not there, it was a question if I should add it
fine for me
cc @p0lyn0mial WDYT?

I think we don't have to because the UI has its own user management module which prevents viewers from editing a node deployment. It was only possible via a direct API call (and if I recall correctly none calls our API directly)

File: CHANGELOG.md (hunk 7 above)
Comment (maciaszczykm) [reply CHANGELOG.md:38]: > The color theme of the Dashboard got redesigned.

Sounds well, I would perhaps change it slightly to:

> The color scheme of the Dashboard was changed.

But it is only the syntax. Meaning is the same :)

File: CHANGELOG.md (hunk 6 above)
Comment (alvaroaleman) [reply CHANGELOG.md:88]: Moved almost everything of it out of the bugfix section, PTAL

**Pull request merged**